packages/analitics/types.ts

```diff
@@ -0,0 +1,40 @@
+import { Package, Action } from "frontity/types";
+import Source from "@frontity/source";
+
+export type Pageview = {
+  page: string;
+  title: string;
+};
+
+export type Event = {
+  category: string;
+  action: string;
+  label: string;
+};
```

Comment:
I'm not sure if we should use `action`, `category` and  `label`. Feels too coupled to Google Analytics. Most analytics tools have something like `name` and `payload` for events.